### PR TITLE
#245 FAQ, DNS resolution

### LINKED ISSUE #245: FAQ Page for the cloud
DNS issues

### PR DESCRIPTION
**Pull request by ylorph**

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -154,6 +154,11 @@ module.exports = {
                         "/cloud/use/",
                         "/cloud/use/tailscale.md"
                     ]
+                },
+                {
+                    title:      "Faq",
+                    collapsable: false,
+                    path:        "/cloud/faq/",
```
Comment (oskardudycz) [inline docs/.vuepress/config.js:171]: **FIX:** We either have to update the styling or add a nested child page. See:
obraz

File: docs/cloud/faq/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# FAQ
+
+## No DNS Resolution of the cluster
+
+Some ISP, router and dns servers will not resolve or filter out answers to DNS queries for  `xxxx.mesdb.eventstore.cloud` because the address resolves to a private IP range.
+
+To check if this is the issue you have, perform the following `nslookup` replacing the domain by your cluster DNS name.
+``` bash
+nslookup buh63kqrh41nfqpviing.mesdb.eventstore.cloud
```
Comment (oskardudycz) [inline docs/cloud/faq/README.md:9]: **CONCERN:** Is `buh63kqrh41nfqpviing` random (not the real cluster DNS name)?

Comment (alexeyzimarev) [reply docs/cloud/faq/README.md:9]: > Some ISP, router and dns servers 
Some Internet providers, routers and DNS servers

Comment (ylorph) [reply docs/cloud/faq/README.md:9]: @oskardudycz  yes buh63kqrh41nfqpviing  is _NOT_ random , but it's in the public DNS system .... so it's discoverable anyway

Comment (oskardudycz) [reply docs/cloud/faq/README.md:9]: I think that it'd be better to randomise it a bit, just in case.

**Review by oskardudycz — APPROVED** [review]

**Review by mat-mcloughlin — APPROVED** [review]

**Pull request merged**